### PR TITLE
report test coverage

### PR DESCRIPTION
**Pull request by westin-m**
In both PR and release builds, report code coverage of Id Web's dll files, excluding test dll files.

### CODE REVIEW

File: build/CodeCoverage.runsettings [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                                <ModulePath>.*\microsoft.identity.web.dll</ModulePath>
+                                <ModulePath>.*\microsoft.identity.web.certificate.dll</ModulePath>
+                                <ModulePath>.*\microsoft.identity.web.certificateless.dll</ModulePath>
+                                <ModulePath>.*\microsoft.identity.web.diagnostics.dll</ModulePath>
+                                <ModulePath>.*\microsoft.identity.web.downstreamapi.dll</ModulePath>
+                                <ModulePath>.*\microsoft.identity.web.graphserviceclient.dll</ModulePath>
+                                <ModulePath>.*\microsoft.identity.web.microsoftgraph.dll</ModulePath>
+                                <ModulePath>.*\microsoft.identity.web.tokenacquisition.dll</ModulePath>
+                                <ModulePath>.*\microsoft.identity.web.tokencache.dll</ModulePath>
+                                <ModulePath>.*\microsoft.identity.web.ui.dll</ModulePath>
```
Comment (jennyf19) [inline build/CodeCoverage.runsettings:34]: do we need ui and ui.views?

Comment (westin-m) [reply build/CodeCoverage.runsettings:34]: Probably not, consider them gone.

File: build/CodeCoverage.runsettings [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Empty "Include" clauses imply all; empty "Exclude" clauses imply none.  
+Each element in the list is a regular expression (ECMAScript syntax). See http://msdn.microsoft.com/library/2k3te2cs.aspx.  
+An item must first match at least one entry in the include list to be included.  
+Included items must then not match any entries in the exclude list to remain included.  
+-->
+
+                        <!-- Match assembly file paths: -->
+                        <ModulePaths>
+                            <Include>
+                                <ModulePath>.*\microsoft.identity.web.dll</ModulePath>
```
Comment (jmprieur) [inline build/CodeCoverage.runsettings:25]: We might want to include only the coverage for the target frameworks that we test?

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @westin-m

File: build/CodeCoverage.runsettings (hunk 2 above)
Comment (westin-m) [reply build/CodeCoverage.runsettings:25]: I removed GraphServiceClient, looks like all of the others have some tests.

**Review by jennyf19 — APPROVED** [review]

**Pull request merged**